server/config/passport/index.js

```diff
@@ -18,7 +18,6 @@ module.exports = (passport) => {
       },
       async (username, password, done) => {
         try {
-          // TODO not working currently
```

Comment:
lovely stuff

Reply:
;D if you would notice correctly, it is red and has been removed ;D